bluesky/callbacks/mpl_plotting.py

```diff
@@ -309,6 +309,12 @@ class LiveGrid(CallbackBase):
 
     ax : Axes, optional
         matplotib Axes; if none specified, new figure and axes are made.
+        
+    axes_positive : list, optional
```

Comment:
Matplotlib also has a parameter to control this called [origin] -- ([example](https://matplotlib.org/2.0.2/examples/pylab_examples/image_origin.html)). We actually use it in this function. This makes me wonder:

1. Should we just expose origin? Do people really need to flip left-to-right or is controlling the vertical sign good enough for most uses?
2. If we do add this separate setting, does it pass the right parameter to origin to ensure the two features don't fight each other in unpredictable ways?

Reply:
@tacaswell is COMPLETELY against using origin, see discussion [here](https://3.basecamp.com/3733838/buckets/4066447/messages/1128589265). It appears that introducing origin means we open ourselves up to some unexpected and weird behaviour. From that discussion it appears that this is the best compromise.

Reply:
OK, I just caught up. Was not aware of this latest opinion on 'origin'. Sounds good.